src/els_code_navigation.erl

```diff
@@ -104,3 +114,18 @@ add_include_uri(#{ id := Id }, Acc) ->
 -spec beginning() -> #{range => #{from => {1, 1}, to => {1, 1}}}.
 beginning() ->
   #{range => #{from => {1, 1}, to => {1, 1}}}.
+
+%% @doc check for a match in any of the module imported functions.
+-spec maybe_imported(uri(), els_dt_document:item(), poi_kind(), any()) ->
+        {ok, uri(), poi()} | {error, not_found}.
+maybe_imported(Uri, Document, function, {F, A}) ->
+  POIs = els_dt_document:pois(Document, [import_entry]),
+  case [{M, F, A} || #{id := {M, FP, AP}} <- POIs, FP =:= F, AP =:= A] of
+    [] -> {error, not_found};
+    [Id|_] ->
+      goto_definition(Uri, #{ kind => application, id => Id
+                            , data => undefined
+                            , range => #{ from => {0, 0}, to => {0, 0}}})
```

Comment:
So this should rather be a recursive call to find, skipping the call to goto_definition. I agree, it felt awkward when I did it.